yt/chyt/controller/internal/jupyt/controller.go

```diff
@@ -123,7 +132,48 @@ func (c *Controller) ParseSpeclet(specletYson yson.RawValue) (any, error) {
 }
 
 func (c *Controller) DescribeOptions(parsedSpeclet any) []strawberry.OptionGroupDescriptor {
-	return nil
+	speclet := parsedSpeclet.(Speclet)
+
+	return []strawberry.OptionGroupDescriptor{
+		{
+			Title: "Jupyt params",
+			Options: []strawberry.OptionDescriptor{
+				{
+					Title:        "Docker image",
+					Name:         "jupyter_docker_image",
+					Type:         strawberry.TypeString,
+					CurrentValue: speclet.JupyterDockerImage,
+					DefaultValue: c.config.PreferredDockerImageOrDefault(),
```

Comment:
If you state that it's a default value, you need to use it in the code as a default value.

And since `JupyterDockerImage` is a `string` now (not a `*string`), the `CurrentValue` will be an empty string, not a `nil`. So UI will not show you that default value either.

Reply:
My bad, it's not the place for default docker image at all. I deleted the default value